roles/kibana/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- import_tasks: manage-environment-spaces.yml
+
+- import_tasks: manage-gateway-logs-index-pattern.yml
```

Comment:
I would rename to `manage-gateway-logs-index-patterns.yml` as it iterates over a list of environments hence plural. Same with `manage-application-logs-index-pattern.yml`